openmc/mgxs/library.py

```diff
@@ -192,11 +192,11 @@ def domain_type(self):
     def domains(self):
         if self._domains == 'all':
             if self.domain_type == 'material':
-                return self.geometry.get_all_materials()
+                return list(self.geometry.get_all_materials().values())
```

Comment:
I re-ran them a few commits ago, but I'll double check again just for good measure.